plugins/microsoft_defender_incidents/plugin.spec.yaml

```
@@ -0,0 +1,655 @@
+      type: integer
+    rbacGroupName:
+      title: Role-Based Access Control Group Name
+      description: The Role-Based Access Control Group associated with the device
+      type: string
+    firstSeen:
+      title: First Seen
+      description: Time when device was first seen
+      type: date
+    healthStatus:
+      title: Health Status
+      description: Health state of the device
+      type: string
+    riskScore:
+      title: Risk Score
+      description: Risk score for the device
+      type: string
+    entities:
+      title: Entities
+      description: All entities that have been identified to be part of, or related to, a given alert
+      type: "[]entity"
+  alert:
+    alertId:
+      title: Alert ID
+      description: Unique identifier to represent the alert
+      type: string
+    serviceSource:
+      title: Service Source
+      description: Service that the alert originates from
+      type: string
+    creationTime:
+      title: Creation Time
+      description: Time when alert was first created
+      type: date
+    lastUpdatedTime:
+      title: Last Updated Time
+      description: Time when alert was last updated at the backend
+      type: date
+    resolvedTime:
+      title: Resolved Time
+      description: Time when alert was resolved
+      type: date
+    firstActivity:
+      title: First Activity
+      description: Time when alert first reported that activity was updated at the backend
+      type: date
+    title:
+      title: Title
+      description: Brief identifying string value available for alert
+      type: string
+    description:
+      title: Description
+      description: String value describing alert
+      type: string
+    status:
+      title: Status
+      description: Categorize alert, where possible values are - New, Active, and Resolved
+      type: string
+    severity:
+      title: Severity
+      description: Indicates the possible impact on assets, where possible values are - Informational, Low, Medium, and High
+      type: string
+    investigationId:
+      title: Investigation ID
+      description: Identifier of investigation triggered by this alert
+      type: integer
+    investigationState:
+      title: Investigation State
+      description: Information on the investigation's current status, where possible values are - Unknown, Terminated, SuccessfullyRemediated, Benign, Failed, PartiallyRemediated, Running, PendingApproval, PendingResource, PartiallyInvestigated, TerminatedByUser, TerminatedBySystem, Queued, InnerFailure, PreexistingAlert, UnsupportedOs, UnsupportedAlertType, and SuppressedAlert
+      type: string
+    actorName:
+      title: Actor Name
+      description: Activity group, if any, the associated with alert
+      type: string
+    threatFamilyName:
+      title: Threat Family Name
+      description: Thread family associated with alert
+      type: string
+    mitreTechniques:
+      title: Mitre Techniques
```

```suggestion
      title: MITRE Techniques
```
This is just company name formatting.